Dockerfile.assisted-service.ocp

```diff
@@ -12,6 +12,9 @@ FROM registry.ci.openshift.org/ocp/4.11:cli AS oc-image
 # Create final image
 FROM registry.ci.openshift.org/ocp/4.11:base
 
+RUN dnf install -y postgresql-server && dnf clean all
```

Comment:
I'm okay with this decision making, but please let's do the following:
* make sure we have a task to handle this in the future
* have a TODO for the relevant line of code, mentioning the above ticket
* update PR description, because currently it doesn't tell enough

Basically any kind of "weird but intended" behavior should be accompanied with those things, so that anyone reading it can have better understanding of the intention

Reply:
Done